$NetBSD: patch-ai,v 1.1.2.2 2009/10/22 14:51:26 tron Exp $

I beleive this test works for all relevant pkgsrc platforms.
It has been similarly transformed upstream after 1.7.0 was released.

--- t/op/io.t.orig	2008-12-18 06:19:20.000000000 +0100
+++ t/op/io.t
@@ -51,28 +51,9 @@ Tests various io opcodes.
     ok(1, 'open with null mode')
 .end
 
-.sub 'tt661_todo_test' :anon
-    # Checks whether the platform is linux, MSWin32, darwin: on other
-    # platforms, the following tests are todo'ed.
-    .include 'sysinfo.pasm'
-    $S0 = sysinfo .SYSINFO_PARROT_OS
-    if $S0 == 'linux' goto tt661_ok
-    if $S0 == 'MSWin32' goto tt661_ok
-    if $S0 == 'darwin' goto tt661_ok
-    if $S0 == 'openbsd' goto tt661_ok
-
-    .return (0)
-
-  tt661_ok:
-    .return (1)
-.end
-
 .include 'iglobals.pasm'
 
 .sub 'open_pipe_for_reading'
-    $I0 = tt661_todo_test()
-    unless $I0 goto open_pipe_for_reading_todoed
-
     .local pmc interp
     interp = getinterp
 
@@ -103,15 +84,9 @@ Tests various io opcodes.
   open_pipe_for_reading_failed:
     nok(1, 'open pipe for reading')
     .return ()
-
-  open_pipe_for_reading_todoed:
-    todo(1, 'Unimplemented in this platform, TT #661')
 .end
 
 .sub 'open_pipe_for_writing'
-    $I0 = tt661_todo_test()
-    unless $I0 goto open_pipe_for_writing_todoed
-
     .local pmc interp
     interp = getinterp
 
@@ -144,9 +119,6 @@ Tests various io opcodes.
   open_pipe_for_writing_failed:
     nok(1, 'open pipe for writing')
     .return ()
-
-  open_pipe_for_writing_todoed:
-    todo(1, 'Unimplemented in this platform, TT #661')
 .end
 
 # Local Variables:
